omise/controllers/front/payment.php

```diff
@@ -0,0 +1,28 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . '/omise/classes/charge.php';
+}
+
```

Comment:
@guzzilar To remove the extending `ModuleFrontController` at the class `Charge`, the command that used to get the PrestaShop context `$this->context` can be changed to `Context::getContext()`.

For example:

Before

```php
$this->context->cart->id_currency
```

After
```php
Context::getContext()->cart->id_currency
```

PrestaShop Reference: [How to access the Context?](http://doc.prestashop.com/display/PS15/Using+the+Context+Object#UsingtheContextObject-HowtoaccesstheContext?)